_posts/2017-01-17-announcing-afip-bill.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Announcing AfipBill"
+date: 2016-12-14 18:06:00
+categories: ["open-source", "rubygems"]
+author: "luciano"
+---
+
+If you live in Argentina and you ever use AFIP, you should already know that their platform is not the best in terms of user friendly.
```

Comment:
@lubc This should be either "... in terms of user friendliness" or "... their platform is not user friendly."